examples/scripts/example-data.mjs

```diff
@@ -33,6 +33,9 @@ fs.readdirSync(`${MAIN_DIR}/src/examples/`).forEach(function (category) {
         exampleData[category][example].javaScriptFunction = Prettier.format(Babel.transform(exampleData[category][example].typeScriptFunction, { retainLines: true, filename: `transformedScript.tsx`, presets: ["typescript"] }).code, { parser: BabelParser.parse, tabWidth: 4 });
         exampleData[category][example].nameSlug = example;
         exampleData[category][example].categorySlug = category;
+        const files = formatters.retrieveStaticObject(exampleFileText, 'FILES');
+        // eslint-disable-next-line no-eval
+        if (files) exampleData[category][example].files = eval('(' + files + ')');
```

Comment:
Yeah, this is the kind of thing that lgtm.com often complains about. Just generally, @ellthompson, it might be worth reviewing the kinds of warnings/errors reporting there regarding the examples browser:
https://lgtm.com/projects/g/playcanvas/engine/alerts/?mode=list (not necessarily for this PR)

Reply:
This code is never executed on the client, only during the built step in a node environment.